pkg/operator/controller/controller.go

```diff
@@ -362,8 +361,8 @@ func (r *reconciler) ensureClusterIngress(ci *operatorv1.IngressController, dnsC
 			errs = append(errs, fmt.Errorf("failed to integrate metrics with openshift-monitoring for clusteringress %s: %v", ci.Name, err))
 		}
 
-		if err := r.syncClusterIngressStatus(deployment, ci); err != nil {
-			errs = append(errs, fmt.Errorf("failed to sync status of clusteringress %s/%s: %v", deployment.Namespace, deployment.Name, err))
+		if err := r.syncIngressControllerStatus(deployment, ci); err != nil {
+			errs = append(errs, fmt.Errorf("failed to sync status of ingresscontroller %s/%s: %v", ci.Namespace, ci.Name, err))
```

Comment:
Nit (and I don't know everyone agrees with me): I would omit ingress controller details from the error message (as the caller already knows the context)